docs/release-notes/artifacts/pr0166.yaml

```
@@ -0,0 +1,16 @@
+# --- Release notes artifact ----
+
+schema_version: 1
+changes:
+  - title: Render retry configuration in template and remove retry_interval attribute
```

A question I had while making artifacts is: Do we want to use code-formatting in the headers of our release notes? Because I'm tempted to suggest:
```suggestion
  - title: Render `retry` configuration in template and remove `retry_interval` attribute
```

Another question I have is, do we want to recommend past tense for headers, like:
```suggestion
  - title: Rendered retry configuration in template and remove retry_interval attribute
```